.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: rust
 rust:
+  - stable
   - nightly
 script:
-  - cargo test
+  - cargo build
+  - |
+    [ $TRAVIS_RUST_VERSION != nightly ] ||
+    cargo test
```

Comment:
Could this just be on the same line, rather than needing the pipe block? ie. `[ $TRAVIS_RUST_VERSION != nightly ] || cargo test`